.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Run Smoke Tests'
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+  workflow_dispatch:
+jobs:
+  smoke_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install operating system dependencies
+        run: npx playwright install-deps
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('package.json', 'yarn.lock') }}
+      - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
```

Comment:
It seems that this is necessary in order to allow executing the next step:
```
❯ npx playwright test --browser all tests/smoke
Please install @playwright/test package to use Playwright Test.
  npm install -D @playwright/test
```

Do we need to install all Sentry dependencies? Could we *just* install `@playwright/test`?